src/Header.js

```diff
@@ -92,11 +92,12 @@ const Header = React.createClass({
         rowHeight = '500px';
       }
       let scrollbarSize = getScrollbarSize() > 0 ? getScrollbarSize() : 0;
+      let updatedWidth = isNaN(this.props.totalWidth - scrollbarSize) ? this.props.totalWidth : this.props.totalWidth - scrollbarSize;
```

Comment:
@malonecj @sonny-chan `Number.isNaN(value)` might have been a marginally more robust approach here. It's not exactly a major issue, but it's worth using the new version wherever we would normally use `isNaN`.

More: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN